server/routes.js

```diff
@@ -240,8 +247,33 @@ function fillAuthorInfo(db) {
 	}
 }
 
+function buildSyndicate(db, num=20) {
+	return (posts) => {
+		var feed = new RSS(config.rssInfo)		
+			
+		for(var i = 0; i < posts.length; i++){		
+			feed.item({		
+				title: posts[i].title.text,		
+				description: md.render(posts[i].content),		
+				url: config.rssInfo.site_url + posts[i].title.url,		
+				guid: posts[i].guid,		
+				categories: posts[i].tags,		
+				author: posts[i].author.displayName,		
+				date: posts[i].timestamp		
+			})		
+		}		
+				
+		return {rss: feed.xml()}	
+	}
+
+}
+
 function aggregatePosts(db, ...pipeline) {
-	return db.posts.aggregate({ $match: { visible: true }}, ...pipeline);
+	return db.posts.aggregate(...pipeline);
+}
+
+function aggregateSafe(db, ...pipeline) {
```

Comment:
So the reason I did `aggregateSafe` was because its aggregation for a non-admin. Right now there's not that much of a difference, but its totally feasible that there could be more important differences for the future. I could rename it to `aggregatePublic` if you'd prefer